docs/sources/tempo/configuration/_index.md

```diff
@@ -319,6 +319,9 @@ metrics_generator:
             # per additional dimension instead of one.
             [enable_client_server_prefix: <bool> | default = false]
 
+            # If enabled another histogram will be produced for interactions over messaging systems middlewares
+            [enable_messaging_system_latency_histogram: <bool> | default = false]
```

Comment:
this latency may be seconds, minutes, hours, days, etc. if someone is interested in this feature over longer time periods they should consider increasing the time that the processor waits to match an edge. default is 10s:

```
metrics_generator:
    processor:
        service_graphs:
            wait: 10s
```

can we note this in the docs?

Reply:
Added a sentence about it.
Although it's relevant for the entire processor... the whole service-graph concept is sensitive to significant time diffs between the spans pairs 🤷🏼‍♂️.
I personally believe that anyone configuring these knobs should be aware of this and understand it at that level... but I suppose another notice won't hurt anybody...

Reply:
Thank you for adding a sentence. It's always good to have an a sentence or two explanation in case someone doesn't know.